packages/core-ethereum/src/account.ts

```diff
@@ -86,7 +88,7 @@ class Account {
           yield false
         }
 
-        ticket = yield false
+        ticket = yield
```

Comment:
intentional?

Reply:
probably not - looked like a typo to me. Also tested the functionality, no difference.